packages/experiments/src/utilities/Slots.tsx

```diff
@@ -0,0 +1,222 @@
+import * as React from 'react';
+// TODO: pull this from utilities instead of adding a dependency to OUFR in Foudnation
+import { memoizeFunction } from 'office-ui-fabric-react';
```

Comment:
Rather than add @uifabric/utilities as dependency to experiments I'll just wait to resolve this TODO upon promotion to Foundation.